pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml

```diff
@@ -4,6 +4,55 @@ layout: none
 theme: none
 content-type: text/ng-template
 ---
+-# TODO: Move this into a separate partial
```

Comment:
it's possible to do this with ng-include but I think it's fine as it is; it's not used anywhere else so there's no code duplication.